hs_core/templates/pages/genericresource.html

```diff
@@ -744,6 +747,11 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                                             title="Unzip here">
                                                         <span class="fa fa-file-archive-o icon-blue"></span>
                                                     </button>
+                                                    <button id="fb-unzip-to-folder" data-fb-action="unzip-to-folder"
+                                                            class="disabled fb-space-right"
```

Comment:
looks like this button remains DISABLED even when it should be enabled
<img width="741" alt="image" src="https://user-images.githubusercontent.com/17934193/191985876-d5eaa2e0-91b6-4818-aa78-5f80ace441dd.png">

Reply:
@devincowan Good catch! Have just pushed a fix. Please check it out and let me know if you see any further issues. Thanks!